src/package_tracking/include/package_tracking/package_edge_detection.h

```diff
@@ -0,0 +1,60 @@
+//
+// Created by Shahwaz Khan on 2/19/21.
+//
+
+//// This node detects the edge of the object which is at the closest distance from the origin of the realsense camera.
+//// The z-axis emerges from the lens of the camera, the y-axis always points downwards and the x-axis towards right.
+//// RIGHT HAND THUMB RULE
+
+#ifndef SRC_PACKAGE_EDGE_DETECTION_H
+#define SRC_PACKAGE_EDGE_DETECTION_H
+
+#include "point_cloud_utilities/pcl_utilities.h"
+
+
+typedef pcl::PointCloud<pcl::PointXYZRGB> PointCloudT;
+
+class PackageTracking{
+
+protected:
+    /* data */
+    std::string file_name_;
+    std::string file_path_ = "/home/shahwaz/catkin_ws/src/my_pcl_tutorials/write_data";
```

Comment:
Path should never be hardcoded to your local computer. Otherwise how would someone else run it.

Reply:
Actually, I don't know how fo give the generic path to the file. Next time show me how to do that or I can searchouline and correct the code.